src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -3033,6 +3034,11 @@ public boolean includeInStatistics()
         return includeInStatistics;
     }
 
+    public FocusManager getFocusManager()
+    {
+        return ServiceUtils.getService(FocusBundleActivator.bundleContext, FocusManager.class);
```

Comment:
would it be a better place for this to be a `static` getter in the `FocusManager` class?

Reply:
I'm deleting this comment. I think it's better to pass instance instead of reaching to a global